### DISCUSSION TITLE
Utility for merging detection results from sliced coco dataset

### DISCUSSION THREAD

**Discussion by rohansaw** [obss/sahi#1015] [Q&A]
I used the coco slicing utility to slice my coco-format dataset into 512x512px patches with an overlap of 0.25. I then used a custom object detector to perform inference on these patches and have the results as a coco-style results json for the patches.

Is there a utility to combine the results, so that the result patches of an image are merged into detection results on a single and can be evaluated against the original coco annotations? I didn't find a direct utility for this in the repository.

Thank you!

**Comment by Trotts** [comment]
Unsure if you were able to sort this, but I had a similar request. See https://github.com/obss/sahi/discussions/1069#discussioncomment-10126945 for my code. 

(sorry I am many months after you asked this -- perhaps it will be useful for people who stumble upon this question)
